wom/src/main/scala/wom/graph/expression/AnonymousExpressionNode.scala

```diff
@@ -14,7 +14,15 @@ object AnonymousExpressionNode {
                                                      expression: WomExpression,
                                                      inputMapping: Map[String, OutputPort],
                                                      constructor: AnonymousExpressionConstructor[T]): ErrorOr[T] = {
-    ExpressionNode.buildFromConstructor(constructor)(identifier, expression, inputMapping)
+    // Anonymous expression nodes are created and then immediately consumed.
+    // Name mangling prevents lookups from finding them and using them after their expiration date.
+    // Naturally, leading underscore is an illegal identifier so would never happen IRL.
```

Comment:
Well... that's true in WDL but I don't think WOM makes any such claim.

OTOH I don't think CWL uses anonymous expressions like this and our n=2 right now, so we should be fine 🤞